docs/en/trusted-solutions/tier1/setup/s3_bucket.md

```diff
@@ -28,107 +30,45 @@ It's possible to start from the [Private S3 bucket AWS CloudFormation template](
   * Accept `I acknowledge that AWS CloudFormation might create IAM resources`
   * Next
 
-## Validation
+### CORS Configuration
+
+CORS policy on your S3 bucket will be minimally configured, allowing only the required origin (your site) and methods, amongst other things.
+
+### IAM Role
+
+The template will create an AWS Role with the following permissions on your bucket:
+
+* Allow Shotgun to access your S3 bucket.
+* Allow access to KMS (if you choose to activate AWS-KMS S3 encryption)
+* Allow the Shotgun account to assume the role by editing the role Trust Relationship.
+
+## Media Isolation Activation
 
 Please contact Shotgun support via the dedicated Slack channel and provide the following information:
   * S3 bucket name
   * AWS Region
   * Shotgun Role ARN
 
-Shotgun will configure your test site to use your own S3 bucket. 
-Once the configuration is done, you can test upload and download of media on your test site.
+Shotgun will configure your test site to use your own S3 bucket.
+
+## Validation
 
-## Manual Steps if needed
+At this stage, you should be able to upload and download media. The Shotgun Transcoding Service should also be able to read, transcode and write back the thumbnails, filmstrip and web friendly versions of your media back to your S3 Bucket. To validate this:
 
-* Create your S3 bucket in your selected region. Please avoid `.` in the bucket name, Shotgun doesn't support them.
-* Configure the bucket Default Encryption, as Shotgun uses the bucket Default Encryption to encrypt new S3 objects.
+1. Log in your Migration Test Site.
+2. From the Navigation Bar, go the the Media app
```

Comment:
Should be just point them to the /help/test route?

Reply:
When we'll have a good name for it, yes.